README.md

```diff
@@ -0,0 +1,10 @@
+**React Native app with firebase notification functionality in android and iOS**
+***
+
+**IMPORTENT:**
+ - Firebase account with gmail: rcbrcb13@gmail.com has been used to configure notifications in android
+ - Mike Chang AuthKey_4972PYBR74 has been used to configure notifications in iOS
```

Comment:
Do not mention any private or confidential information in the Readme of a project hosted on GitHub or any public platform. Save this information in a private document on zoho and provide the link to that document in Readme.